pysilcam/process.py

```diff
@@ -203,119 +163,77 @@ def concentration_check(imbw, settings):
     return sat_check, saturation
 
 
-def get_spine_length(imbw):
-    ''' extracts the spine length of particles from a binary particle image
-    (imbw is a binary roi)
-
-    Args:
-        imbw                : segmented particle ROI (assumes only one particle)
-
-    Returns:
-        spine_length        : spine length of particle (in pixels)
-    '''
-    skel = morphology.skeletonize(imbw)
-    for i in range(2):
-        skel = morphology.binary_dilation(skel)
-    skel = morphology.skeletonize(skel)
-
-    spine_length = np.sum(skel)
-    return spine_length
-
-
-def extract_roi(im, bbox):
-    ''' given an image (im) and bounding box (bbox), this will return the roi
-
-    Args:
-        im                  : any image, such as background-corrected image (imc)
-        bbox                : bounding box from regionprops [r1, c1, r2, c2]
-
-    Returns:
-        roi                 : image cropped to region of interest
-    '''
-    roi = im[bbox[0]:bbox[2], bbox[1]:bbox[3]]  # yep, that't it.
-
-    return roi
-
-
-def measure_particles(imbw, imc, settings, timestamp, nnmodel, class_labels):
+def measure_particles(im_binary, imc, settings, timestamp, nnmodel, class_labels):
     '''Measures properties of particles
 
     Args:
-      imbw (full-frame binary image)
+      im_binary (full-frame binary image)
       imc (full-frame corrected raw image)
       image_index (some sort of tag for location matching)
 
     Returns:
-      stats (list of particle statistics for every particle, according to
-      Partstats class)
+      stats (list of particle statistics for every particle, according to Partstats class)
 
     '''
     # check the converage of the image of particles is acceptable
-    sat_check, saturation = concentration_check(imbw, settings)
-    if (sat_check is False):
+    sat_check, saturation = concentration_check(im_binary, settings)
+    if sat_check == False:
```

Comment:
E712: comparison to False should be 'if cond is False:' or 'if not cond:'  <!-- Automatically posted by Lintly -->